kymatio/scattering2d/scattering2d.py

```diff
@@ -98,10 +98,10 @@ def __init__(self, J, shape, L=8, max_order=2, pre_pad=False):
     def build(self):
         self.M, self.N = self.shape
         self.modulus = Modulus()
-        self.pad = Pad(2**self.J, pre_pad = self.pre_pad)
```

Comment:
So the padding module was able to take a number, but below it seems to be taking a list - was this always possible? Asking because I don't see a modification to the padding module itself

Reply:
it was always allowed by pytorch's standard.